app/components/file_upload_preview/file_upload_preview.js

```diff
@@ -186,5 +203,10 @@ const style = StyleSheet.create({
     scrollViewContent: {
         alignItems: 'flex-end',
         marginLeft: 14
+    },
+    warning: {
+        color: 'white',
```

Comment:
Can we make this to use the error label theme color?

Reply:
@enahum, I followed the error label theme color and it doesn't go well with the dark overlay when selecting images.  I passed this by @asaadmahmood and he gave the ok. We can bring him in this discussion to take a second look